### PR TITLE
Week 8 Lesson 1.7 submission

### PR DESCRIPTION
**Pull request by duokobia**
This is my submission for Week 8.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,66 +2,59 @@ import React, { Fragment, useEffect, useState } from "react";
...
+  }, []);
 
-};
-
-
-function App() {
-
-  const [todoList, setTodoList] = useSemiPersistentState();
+  useEffect(() => {
+    if(isLoading){
```
Comment (kramduckner) [inline src/App.js]: I'll slack you a recording, but this hook isn't working as intended - this conditional statement means we'll only run this hook while the app is loading - but that's not the expected result! I attached a screenshot of the instruction I'm talking about.
Screenshot 2023-03-07 at 9 01 20 AM

**Review by kramduckner — CHANGES_REQUESTED** [review]
Looking great! This is a hard assignment imo. Just one issue I found.

File: src/App.js (hunk 1 above)
Comment (duokobia) [reply src/App.js]: Thanks Mark.

I will check slack for the details.

I appreciate.

Comment (duokobia) [reply src/App.js]: Hi Mark!

I just fixed the bug and changes have been pushed please.

Thanks so much.

I appreciate.

**Comment by duokobia** [comment]
Thanks Mark.

I made the changes already.

I sincerely appreciate.

**Pull request merged**